english/dig/uva/uva.md

```diff
@@ -0,0 +1,15 @@
+You arrive at the Univerity of Virginia and go to room 13 in West Range.
+
+You tap and rap at the door to the chamber.
+
+Nevermore suddenly flies off your shoulder and around the corner. You follow to find
+the bird sitting on the window ledge and tapping at the window's lattice, out of reach.
+
+Satisfied you've returned the bird, you head back to where you exited the marshmallow
+structure.
+
+Do you:
+
+[Walk around the marshmallow structure?](../../explore-outside/investigate-marshmallow-structure/investigate-marshmallow-structure.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.